carstore/bs.go

```diff
@@ -497,14 +500,18 @@ func (cs *CarStore) openNewShardFile(ctx context.Context, user util.Uid, seq int
 
 func (cs *CarStore) writeNewShardFile(ctx context.Context, user util.Uid, seq int, data []byte) (string, error) {
 	// TODO: some overwrite protections
-	fname := filepath.Join(cs.rootDir, fmt.Sprintf("sh-%d-%d", user, seq))
+	fname := filepath.Join(cs.rootDir, fnameForShard(user, seq))
 	if err := os.WriteFile(fname, data, 0664); err != nil {
 		return "", err
 	}
 
 	return fname, nil
 }
 
+func (cs *CarStore) deleteShardFile(ctx context.Context, sh *CarShard) error {
+	return os.Remove(fnameForShard(sh.Usr, sh.Seq))
+}
```

Comment:
ooooo golly deleting from disk makes me nervous, instead of just moving to a "limbo" directory.

Remember this is just repo content, not, eg, images, so very unlikely to contain "hard illegal" content. I guess stuffing in `$bytes` is not unimaginable though.

Reply:
Theres very few reasons we will need to actually hard takedown an account at this level, I think a hard delete of files here is fine, especially since its very easy to re-pull everything back by just un-takedowning the user and triggering an index